scripts/generator/graphql_generator/csharp/ShopifyBuy.cs.erb

```diff
@@ -287,18 +288,24 @@ namespace Shopify.Unity {
         /// </param>
         /// \code
         /// // Example usage querying all products
-        /// ShopifyBuy.Client().products((products, error) => {
+        /// ShopifyBuy.Client().products((products, error, after) => {
         /// 	Debug.Log(products[0].title());
         /// 	Debug.Log(products[1].title());
         /// 	Debug.Log(products.Count);
         /// });
         /// \endcode
-        public void products(ProductsHandler callback, int? first = null, string after = null) {
-            GetProductsList((products, error) => {
+        public void products(ProductsPaginatedHandler callback, int? first = 20, string after = null) {
+            GetProductsList((products, error, afterProductsList) => {
```

Comment:
I'm guessing the variable name here is a result of not wanting to shadow the outer `after`. Minor thing, but maybe it should be `afterGetProductsList` which makes it totally clear what function its basically being re-scoped for.